greenpithumb/humidity_sensor.py

```diff
@@ -0,0 +1,35 @@
+import dht11_exceptions
+
+class HumiditySensor(object):
+	"""Wrapper for humidity sensor."""
+
+	def __init__(self, dht11):
+		"""Creates a new HumiditySensor wrapper.
+
+		Args:
+			dht11: Instance of DHT11 sensor reader class. 
+		"""
+		self._dht11 = dht11
+
+	def get_humidity_level(self):
+		"""Returns humidity level."""
+
+		error_code = self._dht11.read().error_code
```

Comment:
We want to hold onto the result after we read from dht11. This and line 33 are two separate reads, so if we find a zero error code here, the error code for 35's read might not be zero.

So we'd want to save it with something like

```python
dht11_result = self._dht11.read()
```